KUP.Authentication.Api/Controllers/AuthenticationController.cs

```diff
@@ -50,13 +53,6 @@ public IActionResult Get()
                 StringValues xForwardedForValues;
                 HttpContext.Request.Headers.TryGetValue("X-Forwarded-For", out xForwardedForValues);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.k8s.int.kaplan.edu/coding_rules#rule_key=csharpsquid%3AS125)